pages/common/git-count.md

```diff
@@ -0,0 +1,13 @@
+# git count
+
+> Print the total number of all commits.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-count>.
+
+- Print the total number of commits:
```

Comment:
What if we use `display`?
```suggestion
- Display the total number of commits:
```

Reply:
No since it doesn't display, it prints

Reply:
I'd like to really make the standard that we should use 
Display for when it is an interactive log E.G. a man page
and 
Print for when it is just hard printed E.G. echo 1

Reply:
We don't have an standard but IMO `display` is more friendly to new users. In the other hand, I prefer `print` when the output is a not for humans text, like [starship](https://github.com/tldr-pages/tldr/blob/main/pages/common/starship.md)

Reply:
@bl-ue Whats your opinion?

Reply:
I have to reference to @bl-ue's comment [here](https://github.com/tldr-pages/tldr/pull/5969#discussion_r632982394). 

> I'm inclined towards standards for everything common such as this, like you, and in the past I've proposed many such trivial normalizations, but now I think it might be better to just suggest differenting wording on a PR-by-PR basis, because—
> 
> - Not everyone associates the same meaning with the same words,
> - If the pages are more identical terms of wording, they seem more machine generated (which isn't what we want),
> - Because contributors themselves might have something to say about the usage of verbs,
> - A word might be more appropriate for one page but less fit for another, and
> - Lots of standards can intimidate contributors and make contribution more cumbersome than it already is.

I think even if we would find the perfect word in this context, a bit of individuality differentiates our pages from man pages or other documentation. Also, display and print basically mean the same. I get what you mean with display being more interactive, but that's just mere interpretation, and someone else might not agree with that.